fritz

```diff
@@ -1,9 +1,86 @@
 #!/usr/bin/env python
 import argparse
-import sys
-import subprocess
+from contextlib import contextmanager
+import json
 import os
 from pathlib import Path
+import subprocess
+import sys
+
+
+@contextmanager
+def status(message):
+    # stolen from baselayer
+    print(f'[·] {message}', end='')
+    try:
+        yield
+    except:
+        print(f'\r[✗] {message}')
+        raise
+    else:
+        print(f'\r[✓] {message}')
+
+
+def check_configs(cfgs):
+    # use config defaults if configs do not exist
+    for cfg in cfgs:
+        # cwd = str(Path(cfg).parent)
+        c = cfg.replace('.defaults', '')
+        if not Path(c).exists():
+            cd = input(f'{c} does not exist, do you want to use {cfg} (not recommended)? [y/N] ')
+            if cd.lower() == 'y':
+                subprocess.run(["cp", f"{cfg}", f"{c}"])
+            else:
+                raise IOError(f'{c} does not exist, aborting')
+
+
+def get_skyportal_token():
```

Comment:
Fine for now; any reason not to use an API call instead?

Reply:
Blind moment, just ignore.

Reply:
:) Ari helped me with this, so this is what we arrived at after a couple iterations.